cli/module_loader.rs

```diff
@@ -671,13 +729,58 @@ impl ModuleLoader for CliModuleLoader {
     };
     let lib = self.lib;
 
+    // Try to preload code cache for this module.
+    if let Some(cache) = self.shared.code_cache.as_ref().cloned() {
+      let specifier_clone = specifier.clone();
+      let pending_code_cache_fut = spawn_blocking(move || {
+        cache
+          .get_sync(
+            specifier_clone.as_str(),
+            code_cache::CodeCacheType::EsModule,
+          )
+          .map(Cow::from)
+      })
+      .map(|r| r.ok().flatten())
+      .boxed_local();
+      self
+        .pending_code_cache_futs
+        .borrow_mut()
+        .insert(specifier.to_string(), pending_code_cache_fut);
+    }
+
     async move {
       module_load_preparer
         .prepare_module_load(vec![specifier], is_dynamic, lib, root_permissions)
         .await
     }
     .boxed_local()
   }
+
+  fn code_cache_ready(
+    &self,
+    specifier: &ModuleSpecifier,
+    code_cache: &[u8],
```

Comment:
Do we get a slice from V8? Maybe we could change signature in `deno_core` to give us vector directly if that's what we get from v8?

Reply:
Yes, we do get a slice from V8.